### PR TITLE
Feature git graph

### PR DESCRIPTION
**Pull request by Meggie-H**
I have a whole section of commented out code. I apologise for this. Tim and I are still working on fixing it. Just ignore it.

### CODE REVIEW

File: src/components/GitGraph.tsx
Comment (LesediMoko) [inline src/components/GitGraph.tsx:1]: I see that useEffect and useState are imported but not used anywhere. Maybe consider removing them?

File: src/components/ReadMore.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+ 
+const ReadMore = ({ children } : ReadMoreProps) => {
+    const text = children;
+    const [isReadMore, setIsReadMore] = useState(true);
+    const toggleReadMore = () => {
+        setIsReadMore(!isReadMore);
+    };
+    return (
+        <p className="text">
```
Comment (LesediMoko) [inline src/components/ReadMore.tsx]: I'm not sure if ` text ` is a custom style or not (cannot see it being styled anywhere). If it's a type, just remove

**Review by LesediMoko — DISMISSED** [review]
Just a few comments about things that may have been left in by mistake. Other than that, all looks great! Nice use of the switch statement 💥

**Review by yanevdw — APPROVED** [review]
Nicely done!

File: src/components/ReadMore.tsx (hunk 1 above)
Comment (Meggie-H) [reply src/components/ReadMore.tsx]: Please have a look again

**Review by LesediMoko — APPROVED** [review]
Great work, Meg! Looks good to go

**Pull request merged**